docs: update readme for diataxis structure (#2122)
canonical/ubuntu-advantage-client: README.md

![Released Focal Version](https://img.shields.io/ubuntu/v/ubuntu-advantage-tools/focal?label=Focal&logo=ubuntu&logoColor=white)
![Released Jammy Version](https://img.shields.io/ubuntu/v/ubuntu-advantage-tools/jammy?label=Jammy&logo=ubuntu&logoColor=white)

> The Ubuntu Advantage (UA) Client is the offical plataform to enable Canonical offering on your

Reviewer: ```suggestion
The Ubuntu Advantage (UA) Client is the official platform to enable Canonical offerings on your
```

Reviewer: Also "platform" kind of implies something bigger/broader than what we offer I think. Maybe replace "platform" with just "tool"

- [FIPS 140-2 Certified Modules (and optional non-certified patches)](https://ubuntu.com/security/fips)
- [Livepatch Service](https://ubuntu.com/security/livepatch)

> If you need any of those services in your machine, UA is the right tool for you.

Reviewer: ```suggestion
If you need any of those services for your machine, UA is the right tool for you.
```

Furthermore, UA is already installed on every Ubuntu system. Try it out by running `ua help`!

> ## Quickstart
> 
> To learn the basic commands and enable professional Ubuntu services in a sandbox environment, follow our [Getting Started Tutorial](./docs/tutorials/basic_ua_commands.md).

Reviewer: I added this, but now thinking it doesn't add much value over the first entry in the tutorials list. We can just remove it
```suggestion
```
